extension/goext/types.go

```diff
@@ -90,6 +94,19 @@ func (m Maybe) HasValue() bool {
 	return m.MaybeState == MaybeValue
 }
 
+func (m Maybe) String() string {
+	switch m.MaybeState {
+	case MaybeNull:
+		return "<null>"
+	case MaybeUndefined:
+		return "<undefined>"
+	case MaybeValue:
+		return "<valid value>"
```

Comment:
why not just valid?

Reply:
Maybe(String|Int|Float|Bool) also implement String(). They will print 'just value' if they have it. Maybe type doesn't know anything about value, only state.

Reply:
fixed